apps/vr-tests/src/stories/Nav.stories.tsx

```diff
@@ -69,20 +69,22 @@ const disabledLinks: INavLink[] = [
         name: 'MSN',
         url: 'http://msn.com',
         key: 'key2',
-        disabled: true,
-        altText: 'The link is temporary disabled'
+        disabled: true
       }
     ],
     isExpanded: true
   },
-  { name: 'Documents', url: 'http://example.com', key: 'key3' },
-
+  {
+    name: 'Documents',
+    url: 'http://example.com',
+    key: 'key3'
+  },
   {
     name: 'Unavailable Item',
     url: 'http://cnn.com',
-    icon: 'News',
+    icon: IconNames.News,
```

Comment:
You're right about the deprecation of IconNames, good catch. We can change this back to 'News' like you had before.

```suggestion
    icon: 'News',
```

Reply:
The following solution can work for OTB constant icons
https://gist.github.com/winperec/052cba5e7fcb563ab92ccc8ae2f0b1bb 
but I think Office UI Fabric supports registering custom icons and in this case it will not work :)